packages/atomic/src/components/commerce/atomic-commerce-search-box/atomic-commerce-search-box.new.stories.tsx

```diff
@@ -36,6 +36,14 @@ export const RichSearchBox: Story = {
   },
 };
 
+export const StandaloneSearchBox: Story = {
+  name: 'Standalone search box',
+  args: {
+    'attributes-redirection-url':
+      './iframe.html?id=atomic-commerce-search-box--in-page&viewMode=story&args=enable-query-syntax:!true;suggestion-timeout:5000',
+  },
+};
+
```

Comment:
@louis-bompart This is what I was talking about, I added the "attributes-" prefix here, but it does not appear on my atomic-commerce-search-box component. If I don't add the prefix, I do see it, but as just "url" instead of "redirection-url"...